wcfsetup/install/files/lib/form/LostPasswordForm.class.php

```diff
@@ -73,6 +75,10 @@ public function validate()
     {
         parent::validate();
 
+        if (FloodControl::getInstance()->countContent('com.woltlab.wcf.lostPasswordForm', new \DateInterval('PT24H'))['count'] >= self::ALLOWED_RESETS_PER_24H) {
```

Comment:
If I need to scroll horizontally then the line is too long. Keep it at 120 characters, please.